qt/scientific_interfaces/ISISReflectometry/GUI/Runs/QtRunsView.cpp

```diff
@@ -333,6 +342,14 @@ std::string QtRunsView::getSearchString() const {
   return m_ui.textSearch->text().toStdString();
 }
 
+/**
+Get the live data update interval value given by the user.
+@returns The live data update interval
+*/
+std::string QtRunsView::getLiveDataUpdateInterval() const {
```

Comment:
Is there a reason this is converted to a string?

Reply:
Yes, it is to be passed to `alg->setProperty` as a string

Reply:
I guess that's done in the presenter? It's probably better to do the conversion there rather than the view have knowledge about how it will be used.

Reply:
I think you can set properties by value.